helper/ResponseVariableFormatter.php

```diff
@@ -158,14 +161,18 @@ public static function formatStructuredVariablesToItemState(
 
             $itemResults = [];
             foreach ($itemResult['taoResultServer_models_classes_ResponseVariable'] as $var) {
-                /**
-                 * @var $responseVariable \taoResultServer_models_classes_ResponseVariable
-                 */
+                /** @var $responseVariable ResponseVariable */
                 $responseVariable = $var['var'];
 
                 if ($responseVariable->getBaseType() === 'file') {
+                    $file = ['uri' => $var['uri']];
+
+                    if ($rawFile) {
+                        $file = self::formatVariableFile($responseVariable);
+                    }
```

Comment:
There's no reason to have a default `$file` value, I think.

```suggestion
                    $file = $rawFile
                        ? self::formatVariableFile($responseVariable)
                        : ['uri' => $var['uri']];
```

Reply:
No longer required...